frontend/packages/console-dynamic-plugin-sdk/src/extensions/dashboards.ts

```diff
@@ -1,24 +1,35 @@
-import { TFunction } from 'i18next';
```

Comment:
https://github.com/openshift/console/blob/2fdf5791d7d9f238fc6314f95b13c1634fb8072e/frontend/packages/console-dynamic-plugin-sdk/src/extensions/dashboards.ts#L47 can you rename `loader` to `component` ?